src/components/transfers/grid/DeleteAllModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import { useRouter } from 'next/router'
+import { useMutation } from 'react-query'
+import { useTranslation } from 'next-i18next'
+import { AxiosError, AxiosResponse } from 'axios'
+
+import { GridSelectionModel } from '@mui/x-data-grid'
+
+import { ApiErrors } from 'service/apiErrors'
+import { useDeleteManyTransfer } from 'service/transfer'
+
+import { AlertStore } from 'stores/AlertStore'
+import { ModalStore } from 'stores/dashboard/ModalStore'
+
+import { routes } from 'common/routes'
+import { TransferResponse } from 'gql/transfer'
+
+import DeleteAllDialog from 'components/admin/DeleteAllDialog'
+
+export default observer(function DeleteAllModal() {
+  const router = useRouter()
+
+  const { t } = useTranslation('transfer')
+
+  const { hideDeleteAll, selectedIdsToDelete, setSelectedIdsToDelete } = ModalStore
```

Comment:
We don't use the modalstore anymore, please merge `master` branch and see other entities